acceptance/workspace/jobs/create/server.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "method": "POST",
+    "path": "/api/2.1/jobs/create",
+    "response": {
+      "body": {
+        "job_id": 1111
+      }
```

Comment:
Can we have this as a section in test.toml instead?
For example:

```
[[API]]
Path = "POST /api/2.1/jobs/create"
Response.Body = { job_id = 1111 }
```

Reply:
Why in TOML? I'd like it if requests that aren't present in a file are automatically emitted somewhere. If that's a JSON file they can be appended to the list easily.

Reply:
+1, a structured file format like JSON makes recording of the stubs possible. We could do that when we actually run the integration tests.

Reply:
TOML is a structured format as well and we can read and update certain sections and write it back safely.

However, that's out of scope here, I think there many cases where you provide this manually, so it should be optimized for ease of writing manually. We need to discuss/design a bit how that auto recording requests would work.

Why I prefer to use test.toml here is that we'll have config merge where parent directory configs are going to be merged into leaf directory configs. I expect this to be very useful for request configuration, so I'd like them to take advantage of that.